### PR TITLE
Set Up UPP control files for SFS development

### LINKED ISSUE #845: Setup SFS UPP control files
Setup SFS UPP control files (xml and txt) based on GEFS v13 UPP control files.

### LINKED ISSUE #863: Precipitation Bucket
The SFS developer Shan Sun reported that the off-line post output precipitation occurred in 12-hour bucket after F240, despite her having set a 24-hour bucket in the attributes of model output for her SFS experiment. such as
```
[Shan.Sun@hfe09 atmos]$ wgrib2 gfs.t00z.master.grb2f0240 | grep CPRAT
856:50388941:d=1993052100:CPRAT:surface:9-10 day ave fcst:   (this is correct since my archive interval is 24hr)
[Shan.Sun@hfe09 atmos]$ wgrib2 gfs.t00z.master.grb2f0264 | grep CPRAT
856:49907671:d=1993052100:CPRAT:surface:252-264 hour ave fcst: (this is wrong, it should be 10-11 day ave)

```

### PR DESCRIPTION
**Pull request by KarinaAsmar-NOAA**
- Adds SFS control files that generate products from initial version of [SFS list](https://docs.google.com/spreadsheets/d/1lU_ZnY5dyugq9Cs5vy6SDfSMocJjrC2ia2VX8UVkDr4/edit#gid=0) (388 fields). Follows from [UPP issue #845](https://github.com/NOAA-EMC/UPP/issues/845)
- Remove the hard-wired 12-hour bucket for after F240

### CODE REVIEW

**Comment by KarinaAsmar-NOAA** [comment]
@WenMeng-NOAA Please see questions below regarding some of the CPC requested variables:

**200-hPa velocity potential:** UPP currently does not produce this variable, but it is in the NCEP GRIB2 tables as VPOT (Discipline 0, Category 2, Parameter 5). Since this addition requires modifying post_avblflds.xml, should it be included in this PR? 

**subsurface runoff ("drain" in Noah-MP Land Surface Model:** The closest variable found in UPP is an _accumulated_ value (ID 122: Accumulated storm surface runoff). Is this the appropriate field?

**Total sky downward direct radiation at the surface:** UPP does not show any variables of this sort. [NCEP GRIB2 Table 4.2-0-4](https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_table4-2-0-4.shtml) has some options for downward radiation. Would any of those qualify for this request?

The remaining CPC requested variables are in UPP and are being added to the control files: IDs 20 (vertical velocity), 86 (streamfunction), and 162 (helicity).

**Comment by KarinaAsmar-NOAA** [comment]
The following CPC requested variables have been added to the control files and tested: VVEL (id 20), STRM (id 86), HLCY (162).

**Hera**
_standalone test_: /scratch1/NCEPDEV/stmp2/Karina.Asmar/sfs_standalone/post_sfs_1991052100
_regression test_: /scratch1/NCEPDEV/stmp2/Karina.Asmar/regression_tests/rt_upp_sfs

**WCOSS2**
_standalone test_: /lfs/h2/emc/ptmp/karina.asmar/sfs_standalone/post_sfs_1991052100
_regression test_: /lfs/h2/emc/ptmp/karina.asmar/regression_tests/rt_upp_sfs

File: parm/POST_XML_Flat_File_processor_POST_FlatFile.log.20240222
Comment (WenMeng-NOAA) [inline parm/POST_XML_Flat_File_processor_POST_FlatFile.log.20240222:1]: @KarinaAsmar-NOAA This log file shouldn't be committed.

File: parm/POST_XML_Flat_File_processor_POST_FlatFile.log.20240103 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+181417: [00141]  Begin PostXMLPreprocessor
```
Comment (WenMeng-NOAA) [inline parm/POST_XML_Flat_File_processor_POST_FlatFile.log.20240103]: @KarinaAsmar-NOAA This log file shouldn't be committed.

File: parm/POST_XML_Flat_File_processor_POST_FlatFile.log.20240222
Comment (KarinaAsmar-NOAA) [reply parm/POST_XML_Flat_File_processor_POST_FlatFile.log.20240222:1]: @WenMeng-NOAA The log file has been deleted.

File: parm/POST_XML_Flat_File_processor_POST_FlatFile.log.20240103 (hunk 1 above)
Comment (KarinaAsmar-NOAA) [reply parm/POST_XML_Flat_File_processor_POST_FlatFile.log.20240103]: @WenMeng-NOAA The log file has been deleted.

**Comment by KarinaAsmar-NOAA** [comment]
> The following CPC requested variables have been added to the control files and tested: VVEL (id 20), STRM (id 86), HLCY (162).
> 
> **Hera** _standalone test_: /scratch1/NCEPDEV/stmp2/Karina.Asmar/sfs_standalone/post_sfs_1991052100 _regression test_: /scratch1/NCEPDEV/stmp2/Karina.Asmar/regression_tests/rt_upp_sfs
> 
> **WCOSS2** _standalone test_: /lfs/h2/emc/ptmp/karina.asmar/sfs_standalone/post_sfs_1991052100 _regression test_: /lfs/h2/emc/ptmp/karina.asmar/regression_tests/rt_upp_sfs

@WenMeng-NOAA The latest standalone and regression tests with the reverted changes (i.e., removed CPC variables) are in the same paths above.

**Comment by WenMeng-NOAA** [comment]
@KarinaAsmar-NOAA Please sync your branch with the latest UPP develop.

**Comment by KarinaAsmar-NOAA** [comment]
> @KarinaAsmar-NOAA Please sync your branch with the latest UPP develop.

@WenMeng-NOAA The branch should be synced now.

**Comment by WenMeng-NOAA** [comment]
@FernandoAndrade-NOAA You may start the UPP RTs on Hera and Orion. There will no baseline changes expected with this PR.

**Comment by WenMeng-NOAA** [comment]
The UPP RTs were completed on WCOSS2 without changed results.

**Comment by FernandoAndrade-NOAA** [comment]
Hera and hercules RTs completed without changes to results, Orion is rerunning fv3r due to a fatal error opening PRSLEV within that test case.

**Comment by FernandoAndrade-NOAA** [comment]
> @FernandoAndrade-NOAA You may start the UPP RTs on Hera and Orion. There will no baseline changes expected with this PR.

All RTs across Hera, Orion, and Hercules have finished with no changes to results.

**Review by FernandoAndrade-NOAA — APPROVED** [review]

**Review by WenMeng-NOAA — APPROVED** [review]

**Comment by WenMeng-NOAA** [comment]
This PR is ready for merging.

**Pull request merged**